diff -Nuar mysql.orig/dlz_mysql_dynamic.c mysql/dlz_mysql_dynamic.c
--- mysql.orig/dlz_mysql_dynamic.c	2024-08-26 04:10:40.587922704 +0000
+++ mysql/dlz_mysql_dynamic.c	2024-08-26 04:11:16.124578991 +0000
@@ -37,7 +37,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include <mysql/mysql.h>
+#include <mariadb/mysql.h>
 
 #include <dlz_dbi.h>
 #include <dlz_list.h>
diff -Nuar mysql.orig/Makefile mysql/Makefile
--- mysql.orig/Makefile	2024-08-26 04:10:40.587922704 +0000
+++ mysql/Makefile	2024-08-26 04:14:07.171449514 +0000
@@ -26,8 +26,8 @@
 prefix = /usr
 libdir = $(prefix)/lib/bind9
 
-CFLAGS += -fPIC -g -I../include $(shell mysql_config --cflags)
-MYSQL_LIBS=$(shell mysql_config --libs)
+CFLAGS += -fPIC -g -I../include $(shell mariadb-config --cflags)
+MYSQL_LIBS=$(shell mariadb-config --libs)
 
 all: dlz_mysql_dynamic.so
 
