diff -Nuar bind9-9.18.1.orig/debian/changelog bind9-9.18.1/debian/changelog
--- bind9-9.18.1.orig/debian/changelog	2022-03-23 17:48:30.000000000 +0000
+++ bind9-9.18.1/debian/changelog	2022-05-10 10:11:54.387356751 +0000
@@ -1,5 +1,11 @@
 bind9 (1:9.18.1-1ubuntu1) jammy; urgency=medium
 
+  * Enable DLZ plugin and link with mariadb client.
+
+ -- Daekyu Lee <dklee@yidigun.com>  Tue, 10 May 2022 19:11:47 +0900
+
+bind9 (1:9.18.1-1ubuntu1) jammy; urgency=medium
+
   * Merge with Debian unstable (LP: #1965981). Remaining changes:
     - Don't build dnstap as it depends on universe packages:
       + d/control: drop build-depends on libfstrm-dev, libprotobuf-c-dev and
--- bind9-9.18.1.orig/debian/control	2022-03-23 17:48:30.000000000 +0000
+++ bind9-9.18.1/debian/control	2022-05-10 10:27:31.962241306 +0000
@@ -29,7 +29,9 @@
                libxml2-dev,
                pkg-config,
                python3,
-               zlib1g-dev
+               zlib1g-dev,
+               libmariadb-dev,
+               libmariadb-dev-compat
 Build-Depends-Indep: fonts-freefont-otf,
                      latexmk,
                      python3-sphinx,
@@ -54,6 +56,7 @@
          iproute2,
          lsb-base (>= 3.2-14),
          netbase,
+         libmariadb3,
          ${misc:Depends},
          ${shlibs:Depends}
 Suggests: bind-doc,
diff -Nuar bind9-9.18.1.orig/debian/extras/etc/named.conf bind9-9.18.1/debian/extras/etc/named.conf
--- bind9-9.18.1.orig/debian/extras/etc/named.conf	2020-08-25 14:00:13.000000000 +0000
+++ bind9-9.18.1/debian/extras/etc/named.conf	2022-05-10 10:07:35.434711920 +0000
@@ -9,3 +9,4 @@
 include "/etc/bind/named.conf.options";
 include "/etc/bind/named.conf.local";
 include "/etc/bind/named.conf.default-zones";
+include "/etc/bind/named.conf.dlz-mariadb";
diff -Nuar bind9-9.18.1.orig/debian/extras/etc/named.conf.dlz-mariadb bind9-9.18.1/debian/extras/etc/named.conf.dlz-mariadb
--- bind9-9.18.1.orig/debian/extras/etc/named.conf.dlz-mariadb	1970-01-01 00:00:00.000000000 +0000
+++ bind9-9.18.1/debian/extras/etc/named.conf.dlz-mariadb	2022-05-10 10:10:05.732587562 +0000
@@ -0,0 +1 @@
+// need to create this file
diff -Nuar bind9-9.18.1.orig/debian/rules bind9-9.18.1/debian/rules
--- bind9-9.18.1.orig/debian/rules	2022-03-22 19:47:05.000000000 +0000
+++ bind9-9.18.1/debian/rules	2022-05-10 10:08:46.498856616 +0000
@@ -86,6 +86,7 @@
 		--enable-rrl \
 		--enable-filter-aaaa \
 		--disable-native-pkcs11 \
+		--with-dlz-mysql=yes \
 		$(EXTRA_FEATURES)
 
 override_dh_auto_build:
