diff -Nuar bind9-9.16.1.orig/debian/changelog bind9-9.16.1/debian/changelog
--- bind9-9.16.1.orig/debian/changelog	2021-10-27 11:00:32.000000000 +0000
+++ bind9-9.16.1/debian/changelog	2022-02-04 10:24:15.000000000 +0000
@@ -1,3 +1,9 @@
+bind9 (1:9.16.1-0ubuntu2.9) focal; urgency=medium
+
+  * Enable DLZ plugin and link with mariadb client.
+
+ -- Daekyu Lee <dklee@yidigun.com>  Fri, 4 Feb 2021 18:59:33 +0900
+
 bind9 (1:9.16.1-0ubuntu2.9) focal-security; urgency=medium
 
   * SECURITY UPDATE: resolver performance degradation via lame cache abuse
diff -Nuar bind9-9.16.1.orig/debian/control bind9-9.16.1/debian/control
--- bind9-9.16.1.orig/debian/control	2020-09-28 10:27:51.000000000 +0000
+++ bind9-9.16.1/debian/control	2022-02-04 10:24:15.000000000 +0000
@@ -31,7 +31,8 @@
                pkg-config,
                python3,
                python3-ply,
-               zlib1g-dev
+               zlib1g-dev,
+               libmariadbclient-dev
 Standards-Version: 4.1.2
 Vcs-Browser: https://salsa.debian.org/dns-team/bind9
 Vcs-Git: https://salsa.debian.org/dns-team/bind9.git
@@ -48,6 +49,7 @@
          lsb-base (>= 3.2-14),
          iproute2,
          netbase,
+         libmariadb3,
          ${misc:Depends},
          ${shlibs:Depends}
 Suggests: bind-doc,
diff -Nuar bind9-9.16.1.orig/debian/extras/etc/named.conf bind9-9.16.1/debian/extras/etc/named.conf
--- bind9-9.16.1.orig/debian/extras/etc/named.conf	2019-12-17 08:42:05.000000000 +0000
+++ bind9-9.16.1/debian/extras/etc/named.conf	2022-02-04 10:31:28.000000000 +0000
@@ -9,3 +9,4 @@
 include "/etc/bind/named.conf.options";
 include "/etc/bind/named.conf.local";
 include "/etc/bind/named.conf.default-zones";
+include "/etc/bind/named.conf.dlz-mariadb";
diff -Nuar bind9-9.16.1.orig/debian/rules bind9-9.16.1/debian/rules
--- bind9-9.16.1.orig/debian/rules	2021-04-27 11:15:23.000000000 +0000
+++ bind9-9.16.1/debian/rules	2022-02-04 20:42:18.208813200 +0000
@@ -70,6 +70,7 @@
 		--enable-filter-aaaa \
 		--disable-native-pkcs11 \
 		--disable-isc-spnego \
+		--with-dlz-mysql=yes \
 		$(EXTRA_FEATURES)
 
 override_dh_auto_build:
